src/views/artefact-editor/text-side.vue

```diff
@@ -221,11 +243,10 @@ export default class TextSide extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '../../assets./styles/_variables.scss';
```

Comment:
My computer yelled at me that this is invalid (this is on Ubuntu 18.04).  I changed to `@import "@/assets/styles/_variables.scss";` and that seemed to make everyone happy.